src/Common/SqlOptions.cs

```diff
@@ -0,0 +1,127 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.ComponentModel;
+using Microsoft.Azure.WebJobs.Hosting;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+
+namespace Microsoft.Azure.WebJobs.Extensions.Sql
+{
+    /// <summary>
+    /// Represents configuration for <see cref="SqlTriggerAttribute"/>.
+    /// </summary>
+    public class SqlOptions : IOptionsFormatter
+    {
+        public const int DefaultBatchSize = 100;
+        public const int DefaultPollingIntervalMs = 1000;
+        private const int DefaultMinimumPollingIntervalMs = 100;
+
+        // NOTE: please ensure the Readme file and other public documentation are also updated if this value ever
+        // needs to be changed.
+        public const int DefaultMaxChangesPerWorker = 1000;
+        /// <summary>
+        /// Maximum number of changes to process in each iteration of the loop
+        /// </summary>
+        private int _batchSize = DefaultBatchSize;
+        /// <summary>
+        /// Delay in ms between processing each batch of changes
+        /// </summary>
+        private int _pollingIntervalMs = DefaultPollingIntervalMs;
+        private readonly int _minPollingInterval = DefaultMinimumPollingIntervalMs;
+        private int _maxChangesPerWorker = DefaultMaxChangesPerWorker;
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="SqlOptions"/> class.
+        /// </summary>
+        public SqlOptions()
+        {
+
+        }
+
+        /// <summary>
+        /// Gets or sets the number of changes per batch to retrieve from the server.
+        /// The default is 100.
+        /// </summary>
+        public int BatchSize
```

Comment:
We should be using MaxBatchSize (https://github.com/Azure/azure-functions-sql-extension/pull/668).